cmd/generate/codesamples.go

```diff
@@ -25,10 +26,11 @@ var codeSamplesCmd = &model.ExecutableCommand[codeSamplesFlags]{
 	Run:   runCodeSamples,
 	Flags: []flag.Flag{
 		flag.StringFlag{
-			Name:        "schema",
-			Shorthand:   "s",
-			Description: "the schema to generate code samples for",
-			Required:    true,
+			Name:                       "schema",
+			Shorthand:                  "s",
+			Description:                "the schema to generate code samples for",
+			Required:                   true,
+			AutocompleteFileExtensions: []string{".json", ".yaml", ".yml"},
```

Comment:
Probably worth having a global for this like `OpenAPIFileTypes` or whatever

Reply:
I do. I'll just update this to read from that as well.